### PR TITLE
Add codeowners

### PR DESCRIPTION
**Pull request by madil90**
- IsaacYangSLA, yhwen owners for repo
- madil90, YuanTingHsieh owners for test

### CODE REVIEW

**Review by IsaacYangSLA — DISMISSED** [review]

**Comment by IsaacYangSLA** [comment]
CODEOWNERS is required to ensure who can review what portion of codes.

**Review by yhwen — APPROVED** [review]

**Comment by IsaacYangSLA** [comment]
@madil90 , can you also add @nvkevlu to docs folder?

**Comment by IsaacYangSLA** [comment]
@madil90 , all commits have to be signed.  Can you add your GPG and sign this PR?

File: CODEOWNERS
Comment (YuanTingHsieh) [inline CODEOWNERS:2]: do you need an "@" sign before yanchengnv?

Comment (madil90) [reply CODEOWNERS]: Yes. fixed.

**Comment by madil90** [comment]
Contributing guide is being developed.

**Comment by madil90** [comment]
> @madil90 , all commits have to be signed. Can you add your GPG and sign this PR?

signed

File: CONTRIBUTING.md
Comment (IsaacYangSLA) [inline CONTRIBUTING.md:26]: No need to be hurry.  It's better to make sure this file is OK for NVFlare.

Comment (madil90) [reply CONTRIBUTING.md]: Agreed. Fixed.

File: CODE_OF_CONDUCT.md
Comment (YuanTingHsieh) [inline CODE_OF_CONDUCT.md:13]: "pledge to making" -> "pledge to make"

"gender identity and expression" missing comma -> "gender identity, and expression"

File: CODE_OF_CONDUCT.md
Comment (YuanTingHsieh) [inline CODE_OF_CONDUCT.md:30]: others' should be other's ?

File: CODE_OF_CONDUCT.md
Comment (YuanTingHsieh) [inline CODE_OF_CONDUCT.md:45]: "or otherwise" add a comma => "or otherwise,"

Suggest "with regard to" => "about" for simplicity

File: CONTRIBUTING.md
Comment (YuanTingHsieh) [inline CONTRIBUTING.md:18]: suggest "we simply ask" -> "we ask" for simplicity.

suggest "whether you are a seasoned open-source contributor or whether you are a first-time contributor." -> "whether you are a seasoned open-source contributor or a first-time contributor." to remove duplicate words.

File: CONTRIBUTING.md
Comment (YuanTingHsieh) [inline CONTRIBUTING.md:22]: suggest "create an issue discussing" -> "create an issue to discuss"

File: CONTRIBUTING.md
Comment (YuanTingHsieh) [inline CONTRIBUTING.md:151]: "all comments addressed." => "all comments are addressed."

**Review by YuanTingHsieh — CHANGES_REQUESTED** [review]
LGTM in general. Just need to fix some minor grammar issues.

File: CODE_OF_CONDUCT.md
Comment (nvkevlu) [reply CODE_OF_CONDUCT.md:13]: "gender identity and expression" is actually okay here if the expression is in reference to gender

File: CODE_OF_CONDUCT.md
Comment (nvkevlu) [reply CODE_OF_CONDUCT.md:30]: if referring to others as plural, others' can be used

File: CONTRIBUTING.md
Comment (nvkevlu) [inline CONTRIBUTING.md:17]: Should we use NVIDIA FLARE instead of NVFlare?

File: CODE_OF_CONDUCT.md
Comment (nvkevlu) [reply CODE_OF_CONDUCT.md:45]: "otherwise unacceptable" is together

**Review by nvkevlu — APPROVED** [review]

**Review by nvkevlu — COMMENTED** [review]
I think this commit will need to be signed before it will merge.

**Review by YuanTingHsieh — APPROVED** [review]

**Review by IsaacYangSLA — APPROVED** [review]
Let's merge it.

**Review by YuanTingHsieh — APPROVED** [review]

**Pull request merged**
